src/main/io/beeper.c

```diff
@@ -224,7 +224,8 @@ void beeper(beeperMode_e mode)
     if (
         mode == BEEPER_SILENCE || (
             (getBeeperOffMask() & (1 << (BEEPER_USB - 1)))
-            && (batteryConfig()->voltageMeterSource != VOLTAGE_METER_NONE && (getBatteryCellCount() == 0))
+            && batteryConfig()->voltageMeterSource != VOLTAGE_METER_NONE 
```

Comment:
Tghe `batteryConfig()->voltageMeterSource != VOLTAGE_METER_NONE` can be removed now, since the battery state will never get out of `BATTERY_INIT`.

Reply:
Thanks, corrected  in https://github.com/betaflight/betaflight/pull/4121/commits/26ead10f7ce96c7ebc2dd16e1542a6d72709709d

Reply:
All good, we all started by learning GitHub.

Reply:
Hopefully all is ok now, i pushed cherry picked original change because revert of revert looking weird thanks for patience :)